src/Moryx.CommandCenter.Web/src/modules/models/EntryValidation.ts

```diff
@@ -1,12 +1,16 @@
-/*
- * Copyright (c) 2020, Phoenix Contact GmbH & Co. KG
- * Licensed under the Apache License, Version 2.0
-*/
-
-export default class EntryValidation {
-    public minimum: number;
-    public maximum: number;
-    public regex: string;
-    public isRequired: boolean;
-    public isPassword: boolean;
-}
+/*
+ * Copyright (c) 2020, Phoenix Contact GmbH & Co. KG
+ * Licensed under the Apache License, Version 2.0
+*/
+
+export default class EntryValidation {
+    public minimum: number;
+    public maximum: number;
+    public regex: string;
+    public isRequired: boolean;
+    public isPassword: boolean;
+    public deniedValue: string[];
+    public allowedValue: string[];

```

Comment:
For this there is already a possible Values attribute. We should make the generated entry equivalent between using AllowedValues and our Custom PossibleValues